src/t8_unstructured_mesh/README.md

```diff
@@ -0,0 +1,9 @@
+# t8_unstructured_mesh #
+**REMARK: Please note that this feature is still work-in-progress, so a lot of functionality is still missing, but will be added step by step.**
+
+In this folder, we define an unstructured mesh interface.  
+Some application codes are designed for unstructured or uniform meshes and cannot use t8code's tree-based structures directly. For this purpose, an iterable mesh data structure is created as an intermediate level in this folder. This frees the users from having to understand the forest-based concept and increases the usability of the interface. 
```

Comment:
I would not write that they do not have to understand this. This is a more positive formulation I think :)
```suggestion
Some application codes are designed for unstructured or uniform meshes and cannot use t8code's tree-based structures directly. For this purpose, an iterable mesh data structure is created as an intermediate level. This frees the users having to work with the forest-of-trees based concept and increases the usability of the interface. 
```